### PR TITLE
Rearranging editors in editors.json

### PR DESCRIPTION
**Pull request by abchatra**
Rearranged to 
- Live editors first,
- Beta editors next,
- Other editors at the end.

### CODE REVIEW

File: docs/editors.json [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
             "menu": "#F76820",
             "background": "linear-gradient(90deg, #B1E3DA 0%, #E91E63 100%)"
         }
     },
-    "arcadestable": {
-        "name": "Arcade",
+    "microbit": {
+        "name": "micro:bit",
+        "title": "MakeCode for micro:bit",
+        "url": "https://makecode.microbit.org/?github=1",
```
Comment (pelikhan) [inline docs/editors.json:15]: We can drop the GitHub flag now

File: docs/editors.json [hunk 2]
```diff
@@ -1,18 +1,43 @@
 {
     "arcade": {
-        "name": "Arcade BETA",
+        "name": "Arcade",
```
Comment (pelikhan) [inline docs/editors.json:3]: This will break the GitHub explorer which requires beta.

File: docs/editors.json (hunk 1 above)
Comment (abchatra) [reply docs/editors.json]: Removed for microbit.

File: docs/editors.json (hunk 2 above)
Comment (abchatra) [reply docs/editors.json:3]: I don't understand. There is BETA editor too in this list.

Comment (pelikhan) [reply docs/editors.json:3]: The key is used also used as the target id when resolving an editor for a github repo. Once we ship the update, we can have "arcade" point to the released version.

Comment (abchatra) [reply docs/editors.json:3]: Ok fixed, I need to remember to fix this once we relase.

**Comment by shakao** [comment]
looks fine for playground!

**Pull request merged**